docs/completeExamples/completeExamples.xml

```diff
@@ -0,0 +1,485 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<Events xmlns="event-logging:3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="event-logging:3 file://event-logging-v3.0.0.xsd" Version="3.0.0">
+    <Event>
+        <EventTime>
+            <TimeCreated>2017-01-02T03:04:05.678Z</TimeCreated>
+        </EventTime>
+        <EventSource>
+            <System>
+                <Name>FileSharer21</Name>
```

Comment:
Taken the numbers out and replaced with another word.  Hopefully, unlikely to be a "thing"?!